### ISSUE TITLE
Ubuntu won't shut down correctly

### ISSUE THREAD

**Issue by connorourke** [rcasero/doc#5]
I don't suppose anyone has seen this problem, or even better found a fix?

When I try and power off my machine from the Ubuntu side of things by hitting shut down in the top right followed by shutdown in the pop-up,  it just hangs indefinitely.

The desktop is still visible, but the machine is unresponsive. 

Any ideas?

**Comment by connorourke** [comment]
Got it sorted - did two things. 

1) Switched to the nvidia driver

2) added `GRUB_CMDLINE_LINUX_DEFAULT="acpi=noirq quiet splash"` to `/etc/default/grub`

Not sure which fixed it.

**Comment by rcasero** [comment]
It hasn't happened to me, but thanks for reporting it.

**Comment by mantonovic** [comment]
Same problem in my case.. fixed with the nvidia driver only.

**Comment by connorourke** [comment]
Unfortunately that worked only on the first time I shut it down. 

Now I have the same problem - it hangs indefinitely when I try and power off and I have to kill it manually. Very annoying...

Which driver are you using @mantonovic ?

**Comment by mantonovic** [comment]
Hi @connorourke,
I've used that offered by default in "Additional Drivers" in the "Software &S Updates" util window.
The Version is 375.66 from nvidia-755

**Comment by connorourke** [comment]
Thanks. I'll give that one a go.

**Comment by mantonovic** [comment]
hi @connorourke,
a last update about my installation..

connecting a secondary screen on the hdmi port on my usb type c adapter did not work, I had to use the embedded one, and my dell was getting quickly hot, and the fan start to spin loudly.

So I updated the kernel to version 4.13.7 (using "Ukuu" software) and switched back to the nouveau driver. Now everything works perfectly (but without the nvidia gpu) also the htmi port on the usb type c adapter.

best regard
